modules/ROOT/examples/live-demos/footnotes/index.html

```diff
@@ -0,0 +1,27 @@
+<div id="tiny-ui">
+  <textarea id="footnotes" style="width: 100%; height: 500px;">
+    <h2>Welcome to Footnotes</h2>
+    <p>Try this demo of the Footnotes<sup id="footnote_34327763141662096813220" class="mce-footnote"><a href="#footnotes_entry_34327763141662096813220">1</a></sup> Premium Plugin.</p>
+    <ol>
+      <li>Place the insertion point where the new footnote should appear.<sup id="footnote_46102500311662090427490" class="mce-footnote"><a href="#footnotes_entry_46102500311662090427490">2</a></sup></li>
+      <li>Click the <em>Insert Footnote</em> icon in the toolbar, or choose <em>Insert > Insert footnote</em> from the TinyMCE menu.</li>
+      <li>A superscripted footnote number is added to the document.</li>
+      <li>And the insertion point is moved down to a new entry in the footnotes section<sup id="footnote_795387001171662097474091" class="mce-footnote"><a href="#footnotes_entry_795387001171662097474091">3</a></sup> at the bottom of the TinyMCE editor.</li>
+      <li>Add text to this new footnote entry<sup id="footnote_83105724181662097486697" class="mce-footnote"><a href="#footnotes_entry_83105724181662097486697">4</a></sup>.</li>
+    </ol>
+    <p>Each superscripted footnote number links to its respective footnote entry, and each footnote entry links back to its respective superscripted footnote number. This allows for navigation between the editor content and the footnotes section.</p>
```

Comment:
@brian-forte would it be a good idea to move this instructional list out of the demo and into a section on `footnotes.adoc`. This would significantly reduce the size of the demo and improve the description of footnotes on the page.

Reply:
To quote my commit message:

> Material from `/examples/live-demos/footnotes/index.html` edited to work as descriptive material in `footnotes.adoc`. And, so added to `footnotes.adoc`.

Reply:
Also _significantly reduce[d] the size of the demo_.